e2e/test/scenarios/collections/uploads.cy.spec.js

```diff
@@ -0,0 +1,106 @@
+import { restore, queryWritableDB, resyncDatabase } from "e2e/support/helpers";
+
+import { WRITABLE_DB_ID } from "e2e/support/cypress_data";
+
+const FIXTURE_PATH = "../../e2e/support/assets";
+
+const testFiles = [
+  {
+    fileName: "dog_breeds.csv",
+    tableName: "dog_breeds",
+    rowCount: 97,
+  },
+  {
+    fileName: "star_wars_characters.csv",
+    tableName: "star_wars_characters",
+    rowCount: 87,
+  },
+];
+
+describe("CSV Uploading", { tags: ["@external", "@actions"] }, () => {
+  ["postgres"].forEach(dialect => {
+    describe(`CSV Uploading (${dialect})`, () => {
+      beforeEach(() => {
+        restore(`${dialect}-writable`);
+        cy.signInAsAdmin();
+
+        cy.request("POST", "/api/collection", {
+          name: `Uploads Collection`,
+          color: "#000000", // shockingly, this unused field is required
+          parent_id: null,
+        }).then(({ body: { id: collectionId } }) => {
+          cy.wrap(collectionId).as("collectionId");
+        });
+        resyncDatabase({ dbId: WRITABLE_DB_ID });
+        enableUploads();
+      });
+
+      testFiles.forEach(testFile => {
+        it(`Can upload ${testFile.fileName} to a collection`, () => {
+          cy.get("@collectionId").then(collectionId =>
+            cy.visit(`/collection/${collectionId}`),
+          );
+
+          cy.fixture(`${FIXTURE_PATH}/${testFile.fileName}`).then(file => {
+            cy.get("#upload-csv", { force: true }).selectFile(
```

Comment:
I wasn't aware you can use `{ force: true }` in the context of the `.get()` selector.
Are you sure it's not being silently ignored?